src/Tracing/SamplingContext.php

```diff
@@ -48,4 +55,24 @@ public function setParentSampled(?bool $parentSampled): void
     {
         $this->parentSampled = $parentSampled;
     }
+
+    /**
+     * Anything that will be provided as an second argument in {@link startTransaction()} will be set here.
+     *
+     * @param array<string, mixed>|null $additionalContext
+     */
+    public function setAdditionalContext(?array $additionalContext): void
+    {
+        $this->additionalContext = $additionalContext;
+    }
+
+    /**
+     * Anything that was provided as a second argument by {@link startTransaction()} will be returned here.
```

Comment:
I think a better description would be `Gets the additional data that will be provided as a second argument to {@link \Sentry\startTransaction()}.`